core/src/xtdb/db.clj

```diff
@@ -61,7 +61,7 @@
 
 (defprotocol TxLog
   (submit-tx [this tx-events] [this tx-events opts])
-  (open-tx-log ^xtdb.api.ICursor [this after-tx-id]
+  (open-tx-log ^xtdb.api.ICursor [this after-tx-id opts]
```

Comment:
Should we consider this breakage? Is TxLog our own internal SPI not intended for extension outside of XT? 

Any user tx log impls will be broken by an arity change here. One could have overloaded the existing arity, made the second arg after-tx-id _or_ a options map.

@jarohen what do you think?

Reply:
(as discussed on Slack) yep, happy with the breaking change here, let's go for 1.23.0 instead of 1.22.2 and pop a note in the release notes.